### ISSUE TITLE
[Question] Send Firefox into Fullscreen-mode

### ISSUE THREAD

**Issue by Frankstar** [mozilla/geckodriver#492]
Easy Question -> Title says it all - spend the last 2 Days searching for it.
(and pressing F11 ist not an option)

Any Ideas ?

**Comment by andreastt** [comment]
There is a WebDriver command to put the [window into fullscreen](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-fullscreen-window), but it has not been implemented in Marionette yet. You can track https://bugzilla.mozilla.org/show_bug.cgi?id=1189749.

**Comment by Frankstar** [comment]
Since this should be resolved by now i tried it again with Aurora 56.0a1, geckodriver v0.17.0 and selenium 3.4.3
```
geckodriver = "/opt/geckodriver"
capabilities = DesiredCapabilities.FIREFOX
capabilities["marionette"] = True
browser = webdriver.Firefox(capabilities=capabilities, executable_path=geckodriver)
```

`browser.fullscreen()` give me 

> 'WebDriver' object has no attribute 'fullscreen'

so im not sure, is this now a geckodriver issue or selenium didnt implement fullscreen?
